gunicorn/glogging.py

```diff
@@ -109,12 +109,10 @@ def __getitem__(self, k):
             kl = k.lower()
             if kl in self:
                 return super(SafeAtoms, self).__getitem__(kl)
-            else:
```

Comment:
While I prefer your style, there is nothing wrong with this code (and other similar usages of the else clause) and it would be great if we could avoid code churn.